frontend/src/components/core/Text.tsx

```diff
@@ -64,9 +64,8 @@ export const MainHeaderText = styled.h1`
   font-weight: 400;
 `;
 
-export const SecondaryHeaderText = styled.h1`
+export const SecondaryHeaderText = styled(Text)`
   font-size: ${({ theme }) => theme.fontSize.mediumLarge};
-  font-weight: 400;
 `;
```

Comment:
Does this change anything else in the codebase?

Reply:
The only other place it's being used is for the lobby header text and I just double-checked that it doesn't cause anything to change/break, so I think we should be good